src/rust-crypto/rust-crypto.ts

```diff
@@ -206,8 +211,22 @@ export class RustCrypto implements CryptoBackend {
     }
 
     public async exportRoomKeys(): Promise<IMegolmSessionData[]> {
-        // TODO
-        return [];
+        const raw = await this.olmMachine.exportRoomKeys(() => true);
+        return JSON.parse(raw);
+    }
+
+    public async importRoomKeys(keys: IMegolmSessionData[], opts?: ImportRoomKeysOpts): Promise<void> {
+        // TODO when backup support will be added we would need to expose from backup in bindings
```

Comment:
> expose from backup in bindings

I don't understand this. Expose *what* from backup?

Reply:
There is a [`from_backup`](https://github.com/matrix-org/matrix-rust-sdk/blob/a977b3a8d5f3dbd9dcf01196aaaf2d16f9ea64c2/bindings/matrix-sdk-crypto-ffi/src/machine.rs#L1381) flag when importing keys in olmMachine. Used as a reminder to not upload this key to backup later as part of the backup danse.

Reply:
```suggestion
        // TODO when backup support will be added we would need to expose the `from_backup` flag in the bindings
```